dir.traversal.targets

```diff
@@ -112,6 +112,34 @@
     <Message Text="%(PackageIds.Identity)" Importance="High" />
   </Target>
 
+  <Target Name="GetTargetPath" DependsOnTargets="FilterProjects" Returns="@(_ProjectTargetPath)">
+    <SelectTargetPath TargetPaths="@(Project)">
+      <Output TaskParameter="TargetPath" ItemName="_TargetProject"/>
+    </SelectTargetPath>
+    <MSBuild Targets="GetTargetPath"
+             Projects="@(_TargetProject)"
+             BuildInParallel="true">
+      <Output TaskParameter="TargetOutputs" ItemName="_ProjectTargetPath"/>
+    </MSBuild>
+  </Target>
+
+  <UsingTask TaskName="SelectTargetPath" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll">
```

Comment:
We cannot use inline C# tasks as they will blow-up on .NET Core msbuild which doesn't support them.